### DISCUSSION TITLE
how can i get a tweet of my following the public institution ?

### DISCUSSION THREAD

**Discussion by digerati73** [tweepy/tweepy#1191] [Questions] [answered]
i want to get a tweet of my following the public institution  in real time
just like i get a message by push service of server.

the public institution notify of disaster by twitter

and then i want to know the happening of disaster as soon as the public instition notify by twitter.

may i use a tweepy.StreamListener?

or is there any other solution ?

**Comment by Harmon758** [comment, marked as answer]
I'm not sure what "public institution" you're referring to, but yes, you can use a Stream to filter realtime Tweets from a specific user/account.
See https://tweepy.readthedocs.io/en/latest/streaming_how_to.html and https://developer.twitter.com/en/docs/tweets/filter-realtime/api-reference/post-statuses-filter
